tracer/src/Datadog.Trace.BenchmarkDotNet/Datadog.Trace.BenchmarkDotNet.csproj

```diff
@@ -1,10 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <!-- NuGet -->
-    <Version>1.18.4-prerelease</Version>
-    <Title>Datadog APM</Title>
-    <Description>BenchmarkDotNet exporter for Datadog APM</Description>
+    <Version>2.24.0</Version>
+    <Title>Datadog CI Visibility - BenchmarkDotNet</Title>
+    <Description>BenchmarkDotNet exporter for Datadog CI Visibility</Description>
+    <PackageTags>$(PackageTags);BenchmarkDotNet</PackageTags>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
```

Comment:
You shouldn't set this permenantly. It's already conditionally set in `Directory.Build.Props`
```suggestion
```

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3774/commits/5a583c087ff9f055bfb81e21ede8862247db8c3b